src/pages/NewChairConsole.js

```diff
@@ -151,6 +190,34 @@ const NewChairConsole = () => {
     }
   };
 
+  const createElection = (darId, index) => {
+    if (darId !== null) {
```

Comment:
Sorry, forgot to mark this one as well. It's pretty much the same issue as the e !== null check